Makefile

```diff
@@ -6,10 +6,10 @@
 build-all: build-soc build-dgpu
 
 build-soc:
-	./docker-build.sh soc
+	./docker-build.sh soc $(PROXY)
```

Comment:
this doesn't work- as you have both http_Proxy and https_Proxy and they are different

Reply:
I have tested this before PR, so it is working as it is. current change is reflecting what was there on the doc as input for proxy.